app/src/forms/form/exportService.js

```diff
@@ -42,15 +42,49 @@ const EXPORT_FORMATS = Object.freeze({
 
 const service = {
 
+  _readSchemaFields: async (schema) => {
+    const fields = [];
+
+    // function to recursively find all input field names
+    const findFields = (obj) => {
+      // if component is an input field showing in the form,
+      if (obj.input && !obj.hidden) {
+        fields.push(obj.key);
+
+        // if object is a checkbox group
+        if (obj.type && obj.type.includes('checkbox') && obj.values) {
+          // add checkbox fields now
+          obj.values.forEach(element => fields.push(obj.key + '.' + element.value));
```

Comment:
The key seems "ok" for singly nested values, but I'm concerned this will not work for deeper nested situations. May need to think about how this recursion is actually working and its edge cases.